From 3d72f2ddfa4d2cef6645fce7f10f9dea2e0d9a80 Mon Sep 17 00:00:00 2001
From: Alistair Leslie-Hughes <leslie_alistair@hotmail.com>
Date: Fri, 22 Jan 2016 15:35:01 +1100
Subject: [PATCH] user32/edit: WM_PRINTCLIENT should call Paint with NULL
 handle

WM_PRINTCLIENT message, the wParam may contain a valid HDC, 
but Begin/End Draw hasn't been called. So stuffs up drawing.

Signed-off-by: Alistair Leslie-Hughes <leslie_alistair@hotmail.com>
---
 dlls/user32/edit.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/dlls/user32/edit.c b/dlls/user32/edit.c
index ca40ed0..f587b13 100644
--- a/dlls/user32/edit.c
+++ b/dlls/user32/edit.c
@@ -5066,6 +5066,9 @@ LRESULT EditWndProc_common( HWND hwnd, UINT msg, WPARAM wParam, LPARAM lParam, B
 		break;
 
 	case WM_PRINTCLIENT:
+		EDIT_WM_Paint(es, NULL);
+		break;
+
 	case WM_PAINT:
 	        EDIT_WM_Paint(es, (HDC)wParam);
 		break;
-- 
1.9.1

